### ISSUE TITLE
embedded PDF viewer

### ISSUE THREAD

**Issue by ebenenglish** [boston-library/commonwealth-vlr-engine#90]
When I am on the show page for an item
And the primary file for the item is a PDF
I should see an embedded PDF viewer application that allows me to search within and read the content
So that I can view the content without having to download the PDF file

Investigate using [PDF.js](https://mozilla.github.io/pdf.js/) for this.

Example item with PDF as primary file:
```
https://www.digitalcommonwealth.org/search/commonwealth:r494vt25n
```

Direct URL for PDF file:
```
https://bpldcassets.blob.core.windows.net/derivatives/documents/commonwealth:r494vt26x/document_access.pdf
```

First tasks:
* determine what JS libraries/files need to be added to the codebase
* the JS function call to render the viewer with the desired features

**Comment by ebenenglish** [comment]
Example embedded PDF viewer:
* https://newspapers.lib.utah.edu/details?id=21200010

**Comment by ebenenglish** [comment]
Another example, looks to be using PDF.js: https://digitalcommons.framingham.edu/uncategorized/IO_a753cd78-e951-45e7-bfea-698dd0162c2b/ 

Here's an example using the Universal Viewer, though this PDF doesn't seem to be searchable so maybe not a good comparison: https://www.chicopeepubliclibrary.org/archives/items/show/3671

**Comment by ebenenglish** [comment]
Another example, looks like PDF.js in an iframe:
https://ptsongasuml.omeka.net/items/show/11439

**Comment by ebenenglish** [comment]
Also PDF.js inside iframe:
https://openarchives.umb.edu/digital/collection/p15774coll11/id/97

**Comment by ebenenglish** [comment]
Fixed via #121
